theme-components/vertical-full-page-map-alternative-verticals/alternativeverticals.hbs

```diff
@@ -0,0 +1,52 @@
+<div class="yxt-AlternativeVerticals{{#unless isShowingResults}} yxt-AlternativeVerticals--notShowingResults{{/unless}}">
+  <div class="yxt-AlternativeVerticals-noResultsInfo">
+    {{translate phrase='<em class="yxt-AlternativeVerticals-noResultsInfo--emphasized">No results found</em> in [[currentVerticalLabel]].' currentVerticalLabel=currentVerticalLabel escapeHTML=false}}
+    {{#if isShowingResults}}
+      {{translate phrase='Showing <em class="yxt-AlternativeVerticals-noResultsInfo--emphasized">all [[currentVerticalLabel]]</em> instead.' currentVerticalLabel=currentVerticalLabel escapeHTML=false}}
+    {{/if}}
+  </div>
+  {{#if (all verticalSuggestions query)}}
+    <div class="yxt-AlternativeVerticals-suggestionsWrapper">
+      <div class="yxt-AlternativeVerticals-details">
+        {{translate phrase='The following search category yielded results for <span class="yxt-AlternativeVerticals-details--query">"[[query]]"</span>:' pluralForm='The following search categories yielded results for <span class="yxt-AlternativeVerticals-details--query">"[[query]]"</span>:' count=verticalSuggestions.length query=query escapeHTML=false}}
```

Comment:
is it possible to have break up these translate helpers into multiple lines? (also below)

Reply:
@oshi97 Unfortunately, I don't think this is supported in the theme yet! (See SLAP-1178)

Reply:
ah ok got it